From 7caca2f8be471be337f7aea70cd900164bf070eb Mon Sep 17 00:00:00 2001
From: Gokturk Yuksek <gokturk@gentoo.org>
Date: Tue, 15 Aug 2017 21:04:24 -0400
Subject: [PATCH] tsk/img/aff: loop initial declarations are not allowed in C89

Declaring an integer inside a for loop as in for(int i;;) is not
allowed in C89 and causes a build failure. Fix it by declaring the
variable just before the for loop.
---
 tsk/img/aff.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tsk/img/aff.c b/tsk/img/aff.c
index fd9c4ff..a832bfb 100644
--- a/tsk/img/aff.c
+++ b/tsk/img/aff.c
@@ -216,7 +216,8 @@ aff_close(TSK_IMG_INFO * img_info)
 {
     IMG_AFF_INFO *aff_info = (IMG_AFF_INFO *) img_info;
     af_close(aff_info->af_file);
-	for (int i = 0; i < img_info->num_img; i++) {
+	int i;
+	for (i = 0; i < img_info->num_img; i++) {
 		if (img_info->images[i])
 			free(img_info->images[i]);
 	}
-- 
2.10.2

